core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -407,7 +407,7 @@ public function sendTo()
 						$src = rawurldecode($src); // see #3713
 
 						// Embed the image if the URL is now relative
-						if (!preg_match('@^https?://@', $src) && file_exists($this->strImageDir . $src))
+						if (!preg_match('@^https?://@', $src) && ($objFile = new File(StringUtil::stripRootDir($this->strImageDir . $src))) && ($objFile->exists() || $objFile->createDeferredImage()))
```

Comment:
```suggestion
						if (!preg_match('@^https?://@', $src) && ($objFile = new File(StringUtil::stripRootDir($this->strImageDir . $src))) && $objFile->createDeferredImage())
```

Because `File::createDeferredImage()` already calls `$this->exists()`.

BTW, from using the method here, it could also be called `createIfDeferred()` analogous to `createIfNotExists()`.

Reply:
But `createDeferredImage()` would return false, so the `exists()` check is needed here too.
👍 for `createIfDeferred()`